src/libs/scene_utils/README.md

```diff
@@ -0,0 +1,85 @@
+# IoT TwinMaker Scene SDK
+
+## License Notice
+
+Amazon is planning to release this code under an open source license to the general public in the future. As a condition of accessing and using this code prior to its public open source release, you agree that until such public open source release (i) this code is considered as Amazon’s Confidential Information under your MNDA with Amazon and you may not disclose any information about this code or redistribute any portion of this code to any third party, and (ii) your rights to use, copy, and prepare Derivative Works of the code are limited to internal uses only. If you do not agree with these terms, you may not access or use the code. These terms must accompany all copies of the code that you distribute internally until the public open source release. Subject to your compliance with the above terms, this code is provided to you under the terms of the Apache 2.0 license.
+
+## Use
+
+This package provides sample command line scripts that automate the creation of IoT TwinMaker scenes. Upload 3D assets and place them in a 3D scene connected to IoT data via augmented widgets (tag, motion indicator, etc.).
+
+# Samples:
+
+## 1. COOKIE FACTORY `samples/cookie_factory_sample/`
+
+Use the models in the `scenes` directory for the `--assetDirPath` input.
+
+### Example
+
+If running this script from the root of `aws-iot-twinmaker-samples`:
+
+```
+npx ts-node src/libs/scene_utils/samples/cookie_factory_sample/sample.ts --workspaceId [WORKSPACE_ID] --sceneId [SCENE_ID] --assetDirPath src/workspaces/cookiefactory/scenes/
```

Comment:
should this be to a different directory (e.g. left as placeholder similar to the other arguments) ?

Reply:
No, this sample was made specifically for the 3D assets we uploaded in this repo. Before it was in the repo my [instructions](https://code.amazon.com/packages/IotTwinMakerSceneEditor/blobs/mainline/--/README.md) were to copy over the `scenes` directory